app/components/elements/tables/table_body_component.html.erb

```diff
@@ -1,7 +0,0 @@
-<%= tag.tbody class: classes do %>
```

Comment:
Having a separate table body component added unnecessary complexity, especially for testing correctly (since it was passed rows that had been rendered by the table component).